hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java

```diff
@@ -145,6 +146,7 @@ private ClientAuthenticationBuilder getClientAuthentication() throws Exception {
         if (clientAuthType.equals(CLIENT_AUTH_TYPE_SIGNED_JWT)) {
             return new SignedJwtBuilder(new File(jksFilePath), keyAlias, tokenEndpoint, keystorePassword);
         } else if (clientAuthType.equals(CLIENT_AUTH_TYPE_CLIENT_ID_SECRET)) {
+        	logger.debug("Getting token for client: "+clientId);
```

Comment:
Sl4fj log statements support direct sting replacement via {}. I.e. logger.debug("Getting token for client: {}", clientId);

Reply:
updated as advised